### PR TITLE
Feature/add order to work

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
## Changes
- Add position in work model and add ordering in work query.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/work/models.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ class Work(models.Model):
     )
     duration = models.CharField(max_length=100, blank=True)
     location = models.CharField(max_length=500, blank=True)
+    position = models.PositiveIntegerField(blank=True, null=True)
```
Comment (bimal125) [inline apps/work/models.py]: @Rup-Narayan-Rajbanshi  Positions are usually fixed, Enum will be good choice in this case
cc @subinasr

**Review by bimal125 — CHANGES_REQUESTED** [review]

File: apps/work/models.py (hunk 1 above)
Comment (subinasr) [reply apps/work/models.py]: The number of work items can be as many as user wants, how do we work around that for enum fields?

Comment (Rup-Narayan-Rajbanshi) [reply apps/work/models.py]: @bimal125 , i agree with @subinasr

Comment (thenav56) [reply apps/work/models.py]: Maybe the terminology is confusing here? We are talking about the order.
Maybe use `order` or add a **help_text** defining this is used to sort works.

Comment (bimal125) [reply apps/work/models.py]: @Rup-Narayan-Rajbanshi   I  am resolving this, lets's change position to order as @thenav56  mentioned above.

**Review by thenav56 — APPROVED** [review]

**Review by bimal125 — APPROVED** [review]

**Pull request merged**